src/constants/RobotConstants.py

```diff
@@ -0,0 +1,29 @@
+from enum import Enum
+
+
+class RobotConstants:
+  # !
+  maxSpeed = 1
+  scale_speed = 1
+  rotationScale = 0.5
+  rotationCurvature = 2.0
+  rotationDeadzone = 0.075
+
+  isSimulation = False
+
+  period = 0.02
+
+  controller_deadzone = 0.075
+  pilot_controller_id = 0
+  operator_controller_id = 1
+
+  frame_width = 24.5
+  frame_length = 34.5
+
+  light_strip_pwm_port = 1
+
+  class NavXPort(Enum):
+    kUSB = 1
+    kSPI = 2
+
+  navxPort = NavXPort.kUSB
```

Comment:
woah I would not do this. There is already a way to do this: ``wpilib.SPI.Port.kMXP`` or ``wpilib.SPI.Port.kUSB`

Reply:
> woah I would not do this. There is already a way to do this: `wpilib.SPI.Port.kMXP` or `wpilib.SPI.Port.kUSB`

I think you _did_ do it, it's from last year's code ;-)  The constants are actually no longer used (the wpilib constant is used instantiating the gyro), we can remove them.